docs/iot/getting-started/tizen-image-download-flash.md

```diff
@@ -232,7 +232,8 @@ To flash Tizen on your SD card using the IoT Setup Manager:
 
         > **Note**
         >
-        > If the Select Drive does not show anything, detach and insert the SD card again into the computer.
+        > - Currently, the USB SD card reader is only supported, but the internal PC SD card slot is not supported yet.
```

Comment:
Please rephrase as following:
Currently, the USB SD card reader is only supported. The internal PC SD slot is not supported.